localstack/services/awslambda/lambda_api.py

```diff
@@ -1796,6 +1798,23 @@ def _create_response(invocation_result, status_code=200, headers=None):
     elif qualifier and not region.lambdas.get(arn).qualifier_exists(qualifier):
         not_found = not_found_error("{0}:{1}".format(arn, qualifier))
 
+    # remove this block when AWS updates the stepfunctions image to support aws-sdk invocations
+    if not_found and "localstack-internal-awssdk" in arn:
+        # init aws-sdk stepfunctions handler
+        with open(
```

Comment:
nitpick: since we're loading the entire file into memory (via `.read()`), we could also use the `code = load_file(.., mode='rb')` utility here instead of the `with` block.

Reply:
Will do in a quick follow-up with the tests for sync2